connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/UserDefinedFunctionE2ETestSuite.scala

```diff
@@ -388,6 +378,66 @@ class UserDefinedFunctionE2ETestSuite extends QueryTest with RemoteSparkSession
       .head()
     assert(result == 135) // 45 + 90
   }
+
+  test("UDAF custom Aggregator - toColumn") {
+    val session: SparkSession = spark
+    import session.implicits._
+    val aggCol = new CompleteUdafTestInputAggregator().toColumn
+    val ds = spark.range(10).withColumn("extra", col("id") * 2).as[UdafTestInput]
+
+    assert(ds.select(aggCol).head() == 135) // 45 + 90
+    assert(ds.agg(aggCol).head().getLong(0) == 135) // 45 + 90
+  }
+
+  test("UDAF custom Aggregator - multiple extends - toColumn") {
```

Comment:
https://github.com/apache/spark/actions/runs/9957362132/job/27509200696

![image](https://github.com/user-attachments/assets/5b8ae748-9eec-4f68-87bc-29fb6b65d121)


Both tests 'UDAF custom Aggregator - toColumn' and 'UDAF custom Aggregator - multiple extends - toColumn' failed during Maven daily testing. Do you have time to fix them? @xupefei

Reply:
We already landed a fix